packages/actor-init-sparql/lib/ActorInitSparql.ts

```diff
@@ -99,6 +100,11 @@ Options:
       context.queryFormat = args.i;
     }
 
+    // Define the datetime
+    if (args.d) {
+      context.datetime = new Date(args.d);
```

Comment:
I just noticed this, but this has to be set via a namespaced context key, just like `KEY_CONTEXT_SOURCES` below. You could add something like `KEY_CONTEXT_DATETIME` to the memento actor class.

Reply:
fixed.

Reply:
`ActorHttpMemento` also has to be updated to use this key, it will always fail otherwise, as well as the unit tests.

Reply:
oops, amateur move. Now fixed